blocks/ads-block/features/ads/ad-helper.js

```diff
@@ -111,7 +112,7 @@ export const getSectionID = (props) => (
 export const getSlotName = (props = {}) => {
   const { arcSite = '' } = props;
   const { websiteAdPath = '' } = getProperties(arcSite);
-  return `${websiteAdPath}${getSectionID(props)}`;
+  return getAdPath(props) || `${websiteAdPath || ''}${getSectionPath(props) || ''}`;
```

Comment:
@amyerz I also noticed a couple issues here that I didn't notice before:
`getSectionID()` is no longer being used to populate the result of `getSlotName()` and this is a problem because `getSectionID()` is used to ensure that the `slotName` value is consistent with the `section_id` targeting param value set in `setPageTargeting()`. As a result, the `getSlotName()` code also will not include the `websiteAdPath` before the ad-path if populated via `getAdPath()` (which would be a regression). This portion of the code probably needs to be reverted and the changes would be made only in the `formatSectionPath()` function where we can check to see if there _is_ any `sectionPath` and if not, make sure to not add a forward slash to `fmtPath`. Let me know if you have any questions about that.